src/middlewares/officeMiddleware.js

```diff
@@ -0,0 +1,20 @@
+import office from '../db/officeDb'
+
+class officeValidator {
+    static createOffice(req,res,next){
+        try {
+            const {name, type} = req.body
+            if (!name) throw "name field required"
```

Comment:
you shouldn't throw an error, instead, try `return res.status(400).send({message: 'error message'});`

Reply:
the error messages thrown are returned on the catch block as res.status(400).send({message: error})